package.json

```diff
@@ -65,8 +65,7 @@
   },
   "lint-staged": {
     "*.js": [
-      "eslint --format=codeframe --fix",
-      "git add"
+      "eslint --format=codeframe --fix"
```

Comment:
ah gotcha. somehow it seemed like it didn't run for me locally, but i'll check it out